unstructured_ingest/v2/processes/connectors/lancedb/lancedb_local.py

```diff
@@ -0,0 +1,45 @@
+from dataclasses import dataclass
```

Comment:
For consistency, can we remove the `lancedb_` prefix for the files within the lancedb package?